modules/dnn/cmake/download_google_inception5.cmake

```diff
@@ -0,0 +1,30 @@
+function(download_google_inception5 dst_dir status_var)
```

Comment:
`OPENCV_TEST_DATA_PATH` - it must be set
`OPENCV_DNN_TEST_DATA_PATH` - optional, especially in case of embedded device.

> is it possible to move space_shuttle to testdata/dnn

Yes, you can copy this file to opencv_extra.